CHANGELOG.md

```diff
@@ -2,6 +2,90 @@
 
 <!-- next version -->
 
+## 0.111.0
+
+### 💡 Enhancements 💡
+
+- `auto-instrumentation`: set OTEL_LOGS_EXPORTER env var to otlp in python instrumentation (#3330)
+
+- `collector`: Expose the Collector telemetry endpoint by default. (#3361)
+  The collector v0.111.0 changes the default binding of the telemetry metrics endpoint from `0.0.0.0` to `localhost`.
+  To avoid any disruption we fallback to "0.0.0.0:{PORT}" as default address.
+  Details can be found here: [opentelemetry-collector#11251](https://github.com/open-telemetry/opentelemetry-collector/pull/11251)
```

Comment:
```suggestion
- `collector`: Expose the Collector telemetry endpoint by default. (#3361)

  The collector v0.111.0 changes the default binding of the telemetry metrics endpoint from `0.0.0.0` to `localhost`.
  To avoid any disruption we fallback to "0.0.0.0:{PORT}" as default address.
  Details can be found here: [opentelemetry-collector#11251](https://github.com/open-telemetry/opentelemetry-collector/pull/11251)
```